frontend/src/store/Constants.ts

```diff
@@ -1,3 +1,9 @@
 export const FOREST_CLIENT_INPUT_MAX_LENGTH = 8;
 export const FAM_APPLICATION_NAME = 'FAM';
-export const FAM_APPLICATION_ID = 1;
\ No newline at end of file
+export const FAM_APPLICATION_ID = 1;
+export const FAM_ADMIN_ROLE = 'FAM_ADMIN';
+
+export const ErrorDescription = {
+    SelfGrantProhibited: 'Granting admin privilege to self is not allowed.',
+    Default: 'An error has occured.',
+}
```

Comment:
It's minor, but better have this "ErrorDescription" be in where the "ErrorCode" located.